### PR TITLE
feature/PATRO23-307 Remove redundant AppException classes

### PR DESCRIPTION
**Pull request by KlaudiaGajocha**
Redundant AppException classes were removed

### CODE REVIEW

**Comment by IzabelaPalubicka** [comment]
You should move class AppException from user module to shared module (to application project which you have to create)

File: src/modules/budget/api/Intive.Patronage2023.Modules.Budget.Api.csproj [hunk 1]
```diff
@@ -28,6 +28,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\shared\abstractions\Intive.Patronage2023.Shared.Abstractions.csproj" />
     <ProjectReference Include="..\..\..\shared\infrastructure\Intive.Patronage2023.Shared.Infrastructure.csproj" />
+    <ProjectReference Include="..\..\user\application\Intive.Patronage2023.Modules.User.Application.csproj" />
```
Comment (mateuszgiedzinski) [inline src/modules/budget/api/Intive.Patronage2023.Modules.Budget.Api.csproj:31]: this reference is not needed

File: src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj [hunk 2]
```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\shared\abstractions\Intive.Patronage2023.Shared.Abstractions.csproj" />
     <ProjectReference Include="..\..\..\shared\infrastructure\Intive.Patronage2023.Shared.Infrastructure.csproj" />
+    <ProjectReference Include="..\..\user\application\Intive.Patronage2023.Modules.User.Application.csproj" />
```
Comment (mateuszgiedzinski) [inline src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj:32]: this reference is not needed.

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

File: src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj (hunk 2 above)
Comment (KlaudiaGajocha) [reply src/modules/example/api/Intive.Patronage2023.Modules.Example.Api.csproj]: removed

File: src/modules/budget/api/Intive.Patronage2023.Modules.Budget.Api.csproj (hunk 1 above)
Comment (KlaudiaGajocha) [reply src/modules/budget/api/Intive.Patronage2023.Modules.Budget.Api.csproj]: removed

**Review by mateuszgiedzinski — APPROVED** [review]

**Review by IzabelaPalubicka — APPROVED** [review]

**Pull request merged**